lib/queries/availability/index.ts

```diff
@@ -0,0 +1,93 @@
+// import { getBusyVideoTimes } from "@lib/videoClient";
+import { Prisma } from "@prisma/client";
+import dayjs from "dayjs";
+import timezone from "dayjs/plugin/timezone";
+import utc from "dayjs/plugin/utc";
+
+import { asStringOrNull } from "@lib/asStringOrNull";
+import { getWorkingHours } from "@lib/availability";
+import { getBusyCalendarTimes } from "@lib/calendarClient";
+import prisma from "@lib/prisma";
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
```

Comment:
Seems like these plugins aren't being used in this file (`dayjs.tz()` nor `dayjs.utc()`) we  should remove them for now.